manifests/compose/dev/compose.yaml

```diff
@@ -74,6 +74,34 @@ services:
     networks:
       - scaph-network
 
+  node-exporter:
+    image: quay.io/prometheus/node-exporter:latest
+    pid: host
+    ports:
+      - 9100:9100
+    volumes:
+      - type: bind
+        source: /proc
+        target: /host/proc
+      - type: bind
+        source: /sys
+        target: /host/sys
+      - type: bind
+        source: /
+        target: /rootfs
+    command:
+      - --path.procfs=/host/proc
+      - --path.sysfs=/host/sys
+      - --path.rootfs=/rootfs
+      - --collector.rapl  # Enable RAPL collector
+      - --collector.filesystem.mount-points-exclude=^/(sys|proc|dev|host|etc)($$|/)
+    privileged: true
+    user: root
+    cap_add:
+      - ALL
+    networks:
+      - kepler-network
```

Comment:
may be create a different network for node-exporter and add that to prometheus since only prometheus needs to access node-exporter.